knowledge_base/share_files_across_bundles/bundle_with_shared_code/src/notebook.ipynb

```diff
@@ -0,0 +1,42 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "# Traverse to the sync root path.\n",
+    "# The working directory is equal to the directory containing the notebook.\n",
+    "shared_path = os.getcwd() + \"/../../shared\"\n",
+    "\n",
+    "# Add the shared directory to the Python path.\n",
+    "sys.path.append(shared_path)"
```

Comment:
This wouldn't always work since the CWD of notebooks depends on DBR versions. I don't remember all the details, but https://github.com/databricks/bundle-examples/blob/22ebb7080065a7876f80dadca858c27a8da7a7c9/default_python/fixtures/.gitkeep#L11-L17 was a way to portably get the absolute path of a different file.

Now that we are at DBR 15 LTS, perhaps we should start archiving that code though, and just add a comment reference saying something like `when using DBR version XXX, see https://github.com/databricks/bundle-examples/blob/22ebb7080065a7876f80dadca858c27a8da7a7c9/default_python/fixtures/.gitkeep#L11-L17 for how to reference a relative path.`

Reply:
The example uses serverless so it works here.

Agree it's worth calling out that it doesn't work on older DBRs.

Reply:
Added note about DBR or serverless requirement.